modules/issue_tracker/jsx/IssueForm.js

```diff
@@ -312,29 +317,30 @@ class IssueForm extends Component {
           />
           <TextboxElement
             name='PSCID'
-            label='PSCID'
+            label={t('PSCID', {ns: 'loris'})}
             onUserInput={this.setFormData}
             disabled={!hasEditPermission}
             value={this.state.formData.PSCID}
           />
           <TextboxElement
             name='visitLabel'
-            label='Visit Label'
+            label={t('Visit Label', {ns: 'loris'})}
             onUserInput={this.setFormData}
             disabled={!hasEditPermission}
             value={this.state.formData.visitLabel}
           />
           <SelectElement
             name='watching'
-            label='Watching?'
+            label={t('Watching?', {ns: 'issue_tracker'})}
             emptyOption={false}
-            options={{No: 'No', Yes: 'Yes'}}
+            options={{No: 'No',
```

Comment:
The labels (not the backend values) should be translated, "Yes" and "No" already exist in the loris namespace